src/pages/Account/AccountPayment.tsx

```diff
@@ -1,10 +1,96 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Row, Col, Modal, ModalHeader, ModalBody, Alert } from 'reactstrap';
+import { Query } from 'react-apollo';
+import { GET_PAYMENT_SOURCES, PaymentSource } from 'networking/paymentSources';
+import { CreditBalance } from 'networking/users';
 
-class AccountPayment extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Account Payment Page</h1>
-    );
-  }
+import AudioLoading from 'shared/loading/AudioLoading';
+
+import NewCardForm from './NewCardForm';
+import DeleteCardForm from './DeleteCardForm';
+import { AlertProperties } from 'components/work/Alert/Alert';
+
+interface Props extends RouterProps {
+  creditBalance: CreditBalance;
 }
+
+enum ModalType {
+  ADD_NEW_CARD = 'ADD_NEW_CARD',
+  DELETE_CARD = 'DELETE_CARD',
+}
+
+const AccountPayment = ({ creditBalance }: Props) => {
+  const [modal, setModal] = React.useState<ModalType | undefined>(undefined);
+  const [alert, setAlert] = React.useState<AlertProperties>({ msg: '', color: '', show: false });
+  const [paymentSource, setPaymentSource] = React.useState<PaymentSource | undefined>(undefined);
+  
+  return (
+    <Query query={GET_PAYMENT_SOURCES}>
+      {({ loading, error, data }) => {
+        if (loading) {
+          return <AudioLoading height={48} width={96} />;
+        }
+        if (error || !data) {
+          return <h4>{error ? error.message : 'Error / No Data'}</h4>;
+        }
+        const paymentSources = data.getPaymentSources;
+        const renderPaymentSources = paymentSources.map((paymentSource: PaymentSource) => (
+          <ListGroupItem
+            key={paymentSource.id}
+            id={paymentSource.id}
+            className="w-100 d-flex justify-content-between align-items-center"
+          >
+            <h6 className="mb-0">
+              {paymentSource.stripeBrand}&nbsp;(...{paymentSource.stripeLast4})
+            </h6>
+            <i onClick={() => handleModal(ModalType.DELETE_CARD, paymentSource)} className="fas fa-trash-alt" />
+          </ListGroupItem>
+        ));
+
+        return (
+          <section>
+            <Alert color={alert.color} isOpen={!!alert.show} toggle={() => setAlert({ ...alert, show: !alert.show })}>{alert.msg}</Alert>
+
+            <Row>
+              <Col xs="12">
+                <p>Credit Balance: {creditBalance.amountUsd}</p>
+              </Col>
+            </Row>
+            <h4>Credit Cards:</h4>
+            <ListGroup className="mb-2 d-flex flex-column">{renderPaymentSources}</ListGroup>
+
+            <Row>
+              <Col xs="12">
+                <div onClick={() => handleModal(ModalType.ADD_NEW_CARD)} className="w-auto d-inline-block align-items-center">
+                  <i className="fas fa-plus-circle" />
+                  <h6 className="ml-2 mb-0 d-inline-block">Add New Card</h6>
```

Comment:
links should be `a` tag

Reply:
there is no link present

Reply:
shouldn't L64 be an `a` tag vs `div` tag? or a `a` tag inside the `div`?
It is a clickable element (which is a link)

Reply:
are we trying to make just the text clickable or the entire div? if its just the text then i think `a` tag is good, if it;s the entire div we can just wrap the div in a `a` tag

Reply:
if i remember correctly i think react div's have `onClick` method so you might be able to use that instead of a `Link` or `a` tag

Reply:
you shouldn't use div as clickable. it breaks accessible/standards. a tags come with mouse pointer built-in hover states, etc.

Reply:
also to @tommy-bee comment, i think traditionally `a` tags are used to link from page to page
https://www.w3schools.com/tags/tag_a.asp

Reply:
`button` works as modal opener link too and it comes with the built-in mouse pointer states.

Reply:
+1 for button. i think at this point in time since we dont have a full design for the account page yet we can just leave it as a `Button` component and style it later depending on the design

Reply:
ah i see, TIL. Will refrain from putting `onClick` on divs.